File: hotels-backend/Hotels/Models/ReservationService.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Hotels.Models
{
    public partial  class ReservationService
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public decimal Cost { get; set; }
        public int ReservationId { get; set; }
    }
}

File: hotels-client-frontend/src/app/reservation-service.ts
export class ReservationService {
  id: number;
  name: string;
  cost: number;
  reservationId: number;

  constructor() {
    this.id = 0;
    this.name = '';
    this.cost = 0;
    this.reservationId = 0;
  }
}